hal/lib/athena/cpp/priority_mutex.cpp

```diff
@@ -7,6 +7,8 @@
 
 #include "HAL/cpp/priority_mutex.h"
 
+namespace hal {
```

Comment:
You could just do "using namespace hal" here, as everything you're defining is in classes.